postgres/configure-and-start-postgres.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+#
+# Ensures that Postgres is installed on this machine.
+# Creates DB and user.
+# Configures Postgres to accept host connections to new Database.
+
+set -eu
+
+SCRIPT_DIR="$( realpath --strip "$( dirname "${BASH_SOURCE[0]}" )" )"
+source "$(dirname ${SCRIPT_DIR})/common.sh"
+
+############################
+### Arguments processing ###
+############################
+
+usage() {
+    echo "Usage: ${0} --host <HOST> --dbname <DBNAME> --username <USERNAME> \\"
+    echo "            --password <USER_PWD>"
+    echo
+    echo "Options:"
+    echo "   --host <HOST>          Host IP to accept connections on"
+    echo "   --dbname <DBNAME>      Database name to create"
+    echo "   --username <USERNAME>  Role name to create"
+    echo "   --password <USER_PWD>  Password to use for new user"
+    exit 1
+}
+
+HOST=
+DBNAME=
+USERNAME=
+PASSWORD=
+
+# Let's get the command line arguments.
+while [ $# -gt 0 ]; do
+    if [ "${1}" = "--host" ]; then
+        shift; if [ -z "${1:-}" ]; then usage; fi
+        HOST="${1}"
+        shift; continue
+    fi
+    if [ "${1}" = "--dbname" ]; then
+        shift; if [ -z "${1:-}" ]; then usage; fi
+        DBNAME="${1}"
+        shift; continue
+    fi
+    if [ "${1}" = "--username" ]; then
+        shift; if [ -z "${1:-}" ]; then usage; fi
+        USERNAME="${1}"
+        shift; continue
+    fi
+    if [ "${1}" = "--password" ]; then
+        shift; if [ -z "${1:-}" ]; then usage; fi
+        PASSWORD="${1}"
+        shift; continue
+    fi
+    echo -e "\nParameter '${1}' is not recognized\n"
+    usage
+done
+
+if [ -z "${HOST}" ] || [ -z "${DBNAME}" ] || [ -z "${USERNAME}" ] || [ -z "${PASSWORD}" ]; then
+    usage
+fi
+
+
+#####################################
+### Actual installation procedure ###
+#####################################
+
+${SCRIPT_DIR}/install.sh
```

Comment:
Does the start script need to run install? If so, could it check if it's already installed? I could imagine a situation where we'd want to be able to start postgres without an internet connection, and `apt-get update` would fail in that case.